README.md

```diff
@@ -24,6 +24,24 @@ One could also use a standard Clang installation, build a sysroot from the
 sources mentioned above, and compile with
 "--target=wasm32-wasi --sysroot=/path/to/sysroot".
 
+## Install
+
+A typical installation from the release binaries might look like the following:
+```shell script
+wget https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-[version]/wasi-sdk-[version]-linux.tar.gz
+tar xvf wasi-sdk-[version]-linux.tar.gz
+sudo cp -r wasi-sdk-[version] /opt/wasi-sdk
+```
+
+## Use
+
+Use the clang installed in the wasi-sdk directory:
+```shell script
+PATH="/opt/wasi-sdk/bin:${PATH}"
+clang foo.c -o foo.wasm
```

Comment:
The more explict version would be something like :  

```
CC="/opt/wasi-sdk/bin/clang --sysroot=/opt/wasi-sdk/share/wasi-sysroot`"
$CC foo.c -o foo.wasm
```

Reply:
Ok, take a look now; I made it more explicit.